catalog/ui/search-ui/standard/pom.xml

```diff
@@ -253,7 +253,10 @@
         <profile>
             <id>documentation</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>skipDocs</name>
+                    <value>!true</value>
```

Comment:
It has to be falsy. If it's `false` then it won't run if it's undefined (the expected behavior being, if one runs a `mvn clean install` this profile will be run). 
`false` would only run when `-DskipDocs=false` is provided.